de.metas.business/src/main/java/org/adempiere/impexp/bpartner/BPartnerImportTableSqlUpdater.java

```diff
@@ -196,6 +202,27 @@ private void dbUpdateGreetings(final String whereClause)
 		logger.info("Invalid Greeting={}", no);
 	}
 
+	private void dbUpdateJobs(final String whereClause)
+	{
+		StringBuilder sql;
+		int no;
+		sql = new StringBuilder("UPDATE I_BPartner i "
```

Comment:
is there a unique constraint in `C_Job` that makes sure we can have just one matching C_Job record?

Reply:
I think that should not happen, but is possible, so we should prevent.

Reply:
thx, but what i had im mind was not to get *some* `C_Job` but rather add a UC to the C_Job table.
Maybe you can add one on `(Name, AD_Org_ID)` and then also filter for I_BPartner.AD_Org_ID if that makes sense